code/modules/mapping/mapping_helpers.dm

```diff
@@ -257,118 +257,126 @@
 		log_mapping("[src] spawned outside of mapload!")
 		return
 
-	var/obj/machinery/power/apc/apc = locate(/obj/machinery/power/apc) in loc
-	if(!apc)
+	var/obj/machinery/power/apc/target = locate(/obj/machinery/power/apc) in loc
+	if(isnull(!target))
```

Comment:
no, just:

```dm
if(isnull(target))
```

this will do some jank ass shit and fill up mapping logs to high hell